### PR TITLE
fix: vertical border theme is not applied

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* The themes of vertical border and last cell border are not applied to grid.
* wrong calculated `tui-grid-scrollbar-frozen-border` width
스크린샷 2020-05-06 오후 7 07 19

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/examples/example07-themes.html [hunk 1]
```diff
@@ -85,16 +76,22 @@ <h4>Custom options</h4>
   cell: {
     normal: {
       background: '#fbfbfb',
-      border: '#e0e0e0'
+      border: '#e0e0e0',
+      showVerticalBorder: true
```
Comment (jung-han) [inline packages/toast-ui.grid/examples/example07-themes.html:80]: 이런 옵션이 있었군요..

File: packages/toast-ui.grid/src/view/bodyArea.tsx [hunk 2]
```diff
@@ -42,6 +42,12 @@ interface StoreProps {
 
 type Props = OwnProps & StoreProps & DispatchProps;
 
+interface AreaStyle {
+  height: number;
+  overflowX?: 'hidden' | 'scroll' | 'auto';
+  overflowY?: 'hidden' | 'scroll' | 'auto';
```
Comment (jung-han) [inline packages/toast-ui.grid/src/view/bodyArea.tsx:48]: 타입 선언해서 사용해도 될 것 같아요!

File: packages/toast-ui.grid/examples/example07-themes.html (hunk 1 above)
Comment (seonim-ryu) [reply packages/toast-ui.grid/examples/example07-themes.html:80]: @jung-han 그리드 디자인 개선할 때 세로 보더가 없는 상태로 시안을 받았었는데 세로 보더를 사용하고 싶은 경우도 있다고 판단해서 옵션을 추가했던 것으로 기억나네요ㅎ

File: packages/toast-ui.grid/examples/example07-themes.html [hunk 3]
```diff
@@ -216,8 +213,9 @@ <h4>Custom options</h4>
 
     const grid = new tui.Grid({
       el: document.getElementById('grid'),
-      scrollX: false,
-      scrollY: false,
+      scrollX: true,
+      // scrollY: false,
```
Comment (seonim-ryu) [inline packages/toast-ui.grid/examples/example07-themes.html:217]: 아예 제거해도 되지 않을까요ㅎ

**Pull request merged**